checkstyle-sonar-plugin/src/main/java/org/sonar/plugins/checkstyle/CheckstyleRulesDefinition.java

```diff
@@ -60,8 +64,38 @@ static void extractRulesData(NewRepository repository, String xmlRulesFilePath,
         ExternalDescriptionLoader.loadHtmlDescriptions(repository, htmlDescriptionFolder);
         try (InputStream resource = CheckstyleRulesDefinition.class
                 .getResourceAsStream("/org/sonar/l10n/checkstyle.properties")) {
-            PropertyFileLoader.loadNames(repository, resource);
+            loadNames(repository, resource);
         }
         SqaleXmlLoader.load(repository, "/com/sonar/sqale/checkstyle-model.xml");
     }
+
+    @VisibleForTesting
+    static void loadNames(NewRepository repository, InputStream stream) {
+        // code taken from https://github.com/SonarSource/sslr-squid-bridge
```

Comment:
please make exact link - with tag and line number.
In javadoc please explain why we copied code.
If we have to copy smth from library, probably we need to signal library that such code should be reusable (this is whole reason of libraries).